k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -23,6 +23,13 @@ date=$(date +%F_%H-%M)
 solution_yaml=${1:-"solution.yaml"}
 nodename=""
 pods_found=0
+size_limit="500MB"
```

Comment:
Need to update usage() also with these new optional params

Reply:
done

Reply:
Ok. Am not getting option to resolve the

Reply:
I see that option, resolving the comment